etc/kayobe/stackhpc.yml

```diff
@@ -145,7 +145,8 @@ stackhpc_kolla_source_version: "stackhpc/{{ openstack_release }}"
 
 # Kolla Ansible source repository.
 stackhpc_kolla_ansible_source_url: "https://github.com/stackhpc/kolla-ansible"
-stackhpc_kolla_ansible_source_version: "stackhpc/{{ openstack_release }}"
+# stackhpc_kolla_ansible_source_version: "stackhpc/{{ openstack_release }}"
+stackhpc_kolla_ansible_source_version: "fix-prometheus"
```

Comment:
Looks like this fix needs to go upstream.

Reply:
https://review.opendev.org/c/openstack/kolla-ansible/+/926198

Reply:
Always good to add a FIXME for these things.

Reply:
done

Reply:
Ready to revert?

Reply:
Yeah, think so